dbpedia/dbPedia_Composers.js

```diff
@@ -36,8 +36,20 @@ function getResults(queryURL, callback) {
                 name = name[0].trim();
             }
             var nationality = $('span[property="dbo:nationality"]').text().trim();
-            var dateOfBirth = $('span[property="dbo:birthDate"]').text().trim();
-            var dateOfDeath = $('span[property="dbo:deathDate"]').text().trim();
+            //to check if only one date is considered
+			//Eg:http://dbpedia.org/page/David_Breeden has two date of death entries
```

Comment:
is the whole file correctly formatted? Because this seems too much indention here and in other places